lib/please_cli/please_cli/config.py

```diff
@@ -867,6 +867,9 @@
                     'staging': {
                         'nix_path_attribute': 'docker',
                     },
+                    'production': {
+                        'nix_path_attribute': 'dockerflow',
```

Comment:
i would also change `nix_path_attribute` to `dockerflow` for staging and testing